js/colourpicker.js

```diff
@@ -873,87 +878,84 @@
+    function rgb2str(rgb, format,alpha,fallback) {
       if (format === undefined) format = "hex";
       if (alpha === undefined) alpha = true;
       if (fallback === undefined) fallback = "hex";
       rgb = correctedRGB(rgb);
       if (alpha===false) rgb.a = false;
       switch (format) {
-        case "rgb":  
+        case "rgb":        
           if (rgb.a===false || rgb.a===1) {
             return "rgb(" + rgb.r + ", " + rgb.g + ", " + rgb.b + ")";
           }else{
-            return "rgba(" + rgb.r + ", " + rgb.g + ", " + rgb.b + ", " + rgb.a + ")";
+           return "rgba(" + rgb.r + ", " + rgb.g + ", " + rgb.b + ", " + rgb.a + ")";
           }
-          break;
-        case "hex": 
+        break;
+        case "hex":        
           if (alpha === false) rgb.a=false;
           return rgb2hex(rgb);
-          break;
-        case "hsl": 
+
+        case "hsl":        
           var hsl = rgb2hsl(rgb);
           if (hsl.a===false || hsl.a===1) {
             return "hsl(" + hsl.h + ", " + hsl.s + ", " + hsl.l + ")";
           }else{
             return "hsla(" + hsl.h + ", " + hsl.s + ", " + hsl.l + ", " + hsl.a + ")";
           }
           break;
-        //return as hsv even if they ask for hsb for some reason
-        case "hsv": case "hsb": 
+        // return as hsv even if they ask for hsb for some reason
+        case "hsv": case "hsb":        
           var hsb = rgb2hsb(rgb);
           if (hsb.a===false || hsb.a===1) {
             return "hsv(" + hsb.h + ", " + hsb.s + ", " + hsb.b + ")";
           }else{
             return "hsva(" + hsb.h + ", " + hsb.s + ", " + hsb.b + ", " + hsb.a + ")";
           }
           break;
-        case "name": 
-          nm = rgb2name(rgb);
+        case "name":        
+          var nm = rgb2name(rgb);
           if (nm[0]==="#") {
-            //we have a hex string - it wasn't in the names list
-            if (fallback !== "rgb" && fallback !== "hsl" && fallback !== "hsv") return nm //return hex if hex name or other is fallback
-            return rgb2string(rgb,fallback,true);
-          }
+            // we have a hex string - it wasn't in the names list
+            if (fallback !== "rgb" && fallback !== "hsl" && fallback !== "hsv") return nm; // return hex if hex name or other is fallback
+              return rgb2str(rgb,fallback,true);
+            }
           return nm;
-          break;
-        default: return '';
+          
+          default: return '';
+        }
       }
-    }
-    //functions below are all self-explained converters.
+    // functions below are all self-explained converters.
     // for non-alpha situations, HSL, HSV/B, and RGB all return a=false
     // hex returns hex6 or hex8 based on alpha
 
     function rgb2hex(rgb, force) {
       if (force === undefined) force = 0;
-      rgb = correctedRGB(rgb); 
-      if (!rgb) return "";     
-      var a;
+      rgb = correctedRGB(rgb);      if (!rgb) return "";      var a;
```

Comment:
that's weird. I definitely didn't write it like that... I ran part of the code through a beautifier and now I regret that decision.